### DISCUSSION TITLE
Unabe to open the demo instance

### DISCUSSION THREAD

**Discussion by azzamsa** [IgnisDa/ryot#155] [Q&A] [answered]
Hi.

I would like to try it without any complicated steps. So the demo instant serve this need. But it doesn't load and browser loading bar is stuck.

Is the demo okay in your end?

Thanks.

**Comment by IgnisDa** [comment, marked as answer]
Hi, thanks for bringing this into my notice. It should fixed now.

**Reply by azzamsa** [reply]
It works! Thank you.
